app/bundle/src/main/resources/application.conf

```diff
@@ -1,8 +1,209 @@
 bitcoin-s {
-    network = mainnet
+  network = mainnet # regtest, testnet3, mainnet, signet
```

Comment:
Same here, my understanding is our problem was the uinstall process, that corrupted the configuration of a new installation.

Reply:
No, the uninstall process made it so I wasn't correctly installing any of the changes I made so I wasn't sure what changes I did worked.

ATM we need to se the config here for the windows packaging to work